packages/grpc-js/src/server.ts

```diff
@@ -266,10 +285,39 @@ export class Server {
       return;
     }
 
-    this.http2Server.close((err?: Error) => {
-      this.started = false;
-      callback(err);
+    let pendingChecks = 0;
+    let callbackError: Error | undefined;
```

Comment:
I think we're better off just omitting all of the error handling code from this function. The docs don't say that `Http2Session#close` pass an error to the callback, and the current `tryShutdown` implementation doesn't pass any meaningful errors to its own callback.